docs/src/pages/system/styled/UsingOptions.js

```diff
@@ -36,7 +36,8 @@ const customTheme = createTheme({
 
 const MyThemeComponent = styled('div', {
   // Configure which props should be forwarded on DOM
-  shouldForwardProp: (prop) => prop !== 'color' && prop !== 'variant',
+  shouldForwardProp: (prop) =>
+    prop !== 'color' && prop !== 'variant' && prop !== 'sx',
```

Comment:
Nicely spotted, hadn't noticed.

Nitpick: we could use `includes` if you like (same for the other)

```suggestion
  shouldForwardProp: (prop) => !['color', 'variant', 'sx'].includes(prop)
```

Reply:
I would maybe keep the original code, it's easier to read and understand (considering this is a docs, and there are rather new topics that the developer needs to grasp)